inc/class-shortcode-ui.php

```diff
@@ -244,9 +268,13 @@ public function enqueue() {
 				'insert_content_label'              => __( 'Insert Content', 'shortcode-ui' ),
 			),
 			'nonces'     => array(
-				'preview'        => wp_create_nonce( 'shortcode-ui-preview' ),
+				'wp_rest'        => wp_create_nonce( 'wp_rest' ),
```

Comment:
Yes, we need to make sure the endpoint implements authentication. We'll want to include a `permissions_callback` for each endpoint — and it would be good to write tests around this as well.